source/wp-content/themes/wporg-developer-2023/inc/user-content.php

```diff
@@ -379,6 +380,11 @@ public static function wp_editor_feedback( $comment, $display = 'show', $edit =
 		ob_start();
 		echo "<div id='feedback-editor-{$comment_id}' class='feedback-editor{$class}'{$display}>\n";
 		echo "<form id='feedback-form-{$instance}{$form_type}' class='feedback-form' method='post' action='{$action}' name='feedback-form-{$instance}'>\n";
+		printf(
+			"<label class='screen-reader-text' for='feedback-comment-%s'>%s</label>",
+			esc_attr( $instance ),
```

Comment:
Curious why use `esc_attr` here?

Reply:
Because it's injecting into the `for` HTML attribute, do you think this is incorrect?

Reply:
I'm going to merge to keep things moving, but LMK

Reply:
Oh, actually it's correct! I must have been mistaken and mixed up the order last night. Sorry for any confusion!